extra/import_data.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python3
 
 import os
```

Comment:
most if this is outside of the scope for this feature. for example the "api.py" file is basically empty. I guess the previous programmer meant to do something with this or it is normally part of the jinja2 structure. others are also based on previous programming and I haven't "touched" those files yet in my own modifications.

I might want to do a quick overhaul of all code in another feature or before the next release, but I do not see any technical problems arising with these warnings.

I did have a look at main.py and import_data.py (those are within the scope).
import_data.py --> see earlier comment that was partly deleted, but my answer is still there)
main.py --> login_required is used with a wrapper around it.
main.py --> bare except. I do agree, but do not know what kind op excepts I can expect here... I rather leave it for now
main.py --> == False/True is used, because it is defined exactly like that with "check_if_user_active()". I remember having a lot of trouble there to make it work ;). But I will work on the user ROLES in next feature. Will pick it up there 👍